apps/angular-thirty-seconds/src/app/angular-thirty-seconds/snippet-list/snippet-list.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { SnippetListComponent } from './snippet-list.component';
+import { SnippetDemoComponent } from '../snippet-demo/snippet-demo.component';
+import { CodeDemoModule } from '@codelab/code-demos';
+import { FormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+
+@NgModule({
+  declarations: [SnippetListComponent, SnippetDemoComponent],
+  exports: [SnippetListComponent],
```

Comment:
Do you really need to export this component?

Reply:
Yes